interpreter/cling/lib/Interpreter/IncrementalExecutor.h

```diff
@@ -150,7 +156,7 @@ namespace cling {
     void setExternalIncrementalExecutor(IncrementalExecutor *extIncrExec) {
       m_externalIncrementalExecutor = extIncrExec;
     }
-
+    void setCallbacks(InterpreterCallbacks* callbacks) { m_Callbacks = callbacks; }
```

Comment:
Now that it is settable, don't we have to deal elegantly with the fact that m_Callbacks might not be nullptr when setCallbacks is callled (i.e. should the be
```
  auto tmp = m_Callbacks;
  m_Callbacks = callbacks;
  return m_Callbacks;
```
or should we have a getCallbacks 

and/or something else?

Reply:
Let's not overdesign this - it's an internal interface, and we can add it when we need it. Right now it gets called exactly once.